datacube/drivers/postgis/_core.py

```diff
@@ -74,14 +75,23 @@ def schema_qualified(name: str) -> str:
     return f"{SCHEMA_NAME}.{name}"
 
 
-def _get_quoted_connection_info(connection) -> tuple:
-    db, user = connection.execute(
+def get_connection_info(connection: Connection) -> tuple[str, str]:
+    """
+    Obtain information about an open database connection
+    :param connection: An SQLAlchemy connection
+    :return: A tuple consisting of the database name and the user name of the connection
+    """
+    row = connection.execute(
         text("select quote_ident(current_database()), quote_ident(current_user)")
     ).fetchone()
+    assert (
```

Comment:
The 3-line assert can be avoided by putting the comment first:
```python
    # MyPy doesn't ...
    assert row is not None
```

Reply:
Hadn't even noticed that pre-commit had split that line!  Thanks.